### DISCUSSION TITLE
[Feature] <cross account faces>

### DISCUSSION THREAD

**Discussion by AuZ-ASD** [immich-app/immich#11153] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

enable the option to allow cross account faces so not every family member must name and merge (specialy from babys/kids) of all the family members and friends.
so if I merge all faces i could find of my son my gradma see the this "one" face also on his account and does not need to do the same for her account (because i already did)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5089
